.cirrus.yml

```diff
@@ -1,547 +1,498 @@
-container:
-  image: gcr.io/flutter-cirrus/build-flutter-image:latest
+# CIRRUS CONFIGURATION FILE
 
+# We change Flutter's directory to include a space in its name (see
+# $CIRRUS_WORKING_DIR) so that we constantly test path names with
+# spaces in them.
+
+env:
+  GCLOUD_SERVICE_ACCOUNT_KEY: ENCRYPTED[f12abe60f5045d619ef4c79b83dd1e0722a0b0b13dbea95fbe334e2db7fffbcd841a5a92da8824848b539a19afe0c9fb]
+  #BOT: true
+
+clean_commit_message_script:
+  # Unsetting CIRRUS_CHANGE_MESSAGE and CIRRUS_COMMIT_MESSAGE as they
+  # might include non-ASCII characters which makes Gradle crash.
+  # See: https://github.com/flutter/flutter/issues/24935
+  # TODO(amirha): remove once we've migrated to newer Gradle
+  - echo "$CIRRUS_CHANGE_MESSAGE" > /tmp/cirrus_change_message.txt
+  - echo "$CIRRUS_COMMIT_MESSAGE" > /tmp/cirrus_commit_message.txt
+  - export CIRRUS_CHANGE_MESSAGE=""
+  - export CIRRUS_COMMIT_MESSAGE=""
+
+# LINUX SHARDS
 task:
+  container:
+    image: gcr.io/flutter-cirrus/build-flutter-image:latest
   use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true' && $CIRRUS_PR == ''
   env:
-    # Name the SDK directory to include a space so that we constantly
```

Comment:
Removing this comment will make it easier for someone to accidentally remove the space in a future commit.  This has already happened once (and we regressed path names with spaces because of it), which is why this comment got added here.

Reply:
Noted. I've attempted to make this even more obvious.